content/en/docs/concepts/storage/volumes.md

```diff
@@ -1342,7 +1343,9 @@ spec:
               foo: bar
 ```
 
-This feature requires CSIInlineVolume feature gate to be enabled:
+This feature requires CSIInlineVolume feature gate to be enabled. It
+is enabled by default in Kubernetes >= 1.16. In Kubernetes v1.15, it
```

Comment:
@pohly , is line 1346 required, as the default is enabled for v1.16?

Reply:
Personally I prefer to keep the information about older releases if it doesn't get too long, because then users don't need to run a mental diff against older documentation to figure out the history. Some people have to work with more than just one release of Kubernetes, so that history is relevant.

That this feature gate exists is also relevant, regardless of the history and whether its enabled or disabled by default.

Reply:
@pohly I agree with @kbhawkey  on this. Saying "It
is enabled by default in Kubernetes >= 1.16" already explains it is not enabled by default in previous releases.

Reply:
I assume future releases of the documentation will still have that paragraph. With that in mind I changed the second sentence to "It is enabled by default starting with Kubernetes 1.16.".

The alternative, "It is enabled by default in Kubernetes 1.16", would look weird in documentation for Kubernetes 1.17...

Reply:
@pohly  Thanks for jumping on this. Looks like the change you mentioned _It is enabled by default starting with Kubernetes 1.16_ is not yet in the doc.

Reply:
Oh, sorry, forgot the final `git push`. It's there now.